File: Patterns/2. Rectangle Patterns/20. Rectangle Alternating Symbols/ras.cpp
#include <iostream>
using namespace std;
int main(){
    int l,b;
    cout<<"Enter the Length : ";
    cin>>l;
    cout<<"Enter the Breadth : ";
    cin>>b;

    for(int i=1; i<=l; i++){
        for(int j=1; j<=b; j++){
            if(j%2!=0){
                cout<<"* ";
            }else{
                cout<<"# ";
            }
        }
        cout<<'\n';
    }
}

/*
Output

* # * # * # * 
* # * # * # * 
* # * # * # * 
* # * # * # * 
* # * # * # * 

*/

File: Patterns/2. Rectangle Patterns/20. Rectangle Alternating Symbols/ras.py
l = int(input("Enter the Length : "))
b = int(input("Enter the Breadth : "))

for i in range(1,l+1):
    for j in range(1,b+1):
        if(j%2!=0):
            print("*", end = ' ')
        else:
            print("#", end = ' ')
    print()

'''
Output:

* # * # * # * 
* # * # * # * 
* # * # * # * 
* # * # * # * 
* # * # * # * 
'''